src/simplewallet/simplewallet.cpp

```diff
@@ -4468,6 +4485,23 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
       success_msg_writer() << "Timestamp: " << get_human_readable_timestamp(pd.m_timestamp);
       success_msg_writer() << "Amount: " << print_money(pd.m_amount);
       success_msg_writer() << "Payment ID: " << payment_id;
+      if (pd.m_unlock_time < CRYPTONOTE_MAX_BLOCK_NUMBER)
```

Comment:
Oh, I already used the max with block height, it didn't show on this comment.
Now, there is the possibility that someone locked for a very long time and we just don't know, but we can't without rescanning the chain. I think I don't really care here.
I had however forgot to take CRYPTONOTE_DEFFAULT_TX_SPENDABLE_AGE, which is a bit iffy as it's a wallet thing, not a consensus thing. I added it now.